test/reporters/tests/__snapshots__/reporters.spec.ts.snap

```diff
@@ -64,12 +64,15 @@ exports[`JUnit reporter with outputFile object in non-existing directory 2`] = `
 
 exports[`json reporter (no outputFile entry) 1`] = `
 {
-  "numFailedTestSuites": 0,
+  "numFailedTestSuites": 1,
   "numFailedTests": 1,
-  "numPassedTestSuites": 2,
-  "numPassedTests": 8,
+  "numPassedTestSuites": 1,
+  "numPassedTests": 6,
   "numPendingTestSuites": 0,
   "numPendingTests": 0,
+  "numSkippedTestSuites": 0,
```

Comment:
Adding new fields would be a feature, not a fix. This reporter is meant to be compatible with Jest. Tools don't support this field

Let's not add any new fields